Adapter_Revit_UI/AdapterActions/Push.cs

```diff
@@ -109,11 +108,16 @@ public override List<object> Push(IEnumerable<object> objects, string tag = "",
                 }
                 else if (pushType == PushType.UpdateOnly)
                 {
-                    BH.Engine.Reflection.Compute.RecordError("Update is currently not supported by Revit_Toolkit, please use DeleteThenCreate instead.");
+                    foreach (IBHoMObject obj in objectsToPush)
+                    {
+                        Element element = obj.Element(document);
+                        if (element != null)
+                            success |= Update(element, obj, RevitSettings);
+                    }
                 }
                 else if (pushType == PushType.FullCRUD)
                 {
-                    BH.Engine.Reflection.Compute.RecordError("Full CRUD is currently not supported by Revit_Toolkit, please use DeleteThenCreate instead.");
+                    BH.Engine.Reflection.Compute.RecordError("Full CRUD is currently not supported by Revit_Toolkit, please use Create, UpdateOnly or DeleteThenCreate instead.");
                 }
```

Comment:
This could be moved up around line 63, because it's a blocking check.

Reply:
I moved the blocking check as far up as possible, should be fine now